src/_nebari/upgrade.py

```diff
@@ -629,27 +650,93 @@ def _version_specific_upgrade(
-            ""
-        )
+        if kwargs.get("attempt_fixes", False):
+            try:
+                kubernetes.config.load_kube_config()
+            except kubernetes.config.config_exception.ConfigException:
+                rich.print(
+                    "[red bold]No default kube configuration file was found. Make sure to [link=https://www.nebari.dev/docs/how-tos/debug-nebari#generating-the-kubeconfig]have one pointing to your Nebari cluster[/link] before upgrading.[/red bold]"
+                )
+                exit()
 
-        continue_ = Prompt.ask(
-            "Have you deleted the Argo Workflows CRDs and service accounts? [y/N] ",
-            default="N",
-        )
-        if not continue_ == "y":
+            for crd in argo_crds:
+                api_instance = kubernetes.client.ApiextensionsV1Api()
+                try:
+                    api_instance.delete_custom_resource_definition(
+                        name=crd,
+                    )
+                except kubernetes.client.exceptions.ApiException as e:
+                    if e.status == 404:
+                        rich.print(f"CRD [yellow]{crd}[/yellow] not found. Ignoring.")
+                    else:
+                        raise e
+                else:
+                    rich.print(f"Successfully removed CRD [green]{crd}[/green]")
+
+            for sa in argo_sa:
+                api_instance = kubernetes.client.CoreV1Api()
+                try:
+                    api_instance.delete_namespaced_service_account(
+                        sa,
+                        namespace,
+                    )
+                except kubernetes.client.exceptions.ApiException as e:
+                    if e.status == 404:
+                        rich.print(
+                            f"Service account [yellow]{sa}[/yellow] not found. Ignoring."
+                        )
+                    else:
+                        raise e
+                else:
+                    rich.print(
+                        f"Successfully removed service account [green]{sa}[/green]"
+                    )
```

Comment:
@marcelovilla I was able to adapt [what you did](https://github.com/smokestacklightnin/nebari/blob/88be8a4db8f2e4b6ae2dde6baf24f7c563f66d56/src/_nebari/upgrade.py#L1154-L1181)